package.json

```diff
@@ -48,7 +48,7 @@
     "chalk": "^1.0.0",
     "core-object": "^1.1.0",
     "ember-cli-babel": "^5.0.0",
-    "ember-cli-deploy-plugin": "^0.2.1",
+    "ember-cli-deploy-plugin": "^0.2.9",
```

Comment:
I've been bitten by loose versioning multiple times now. I'm not sure but isn't it better to use a strict version number here. Its ok not to also, perhaps the pros out weigh the cons. Just wanted to pick your brains.